core/modules/layout/layout.admin.inc

```diff
@@ -281,23 +281,57 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     }
   }
 
-  $items = array(
-    t('Content: <code>node/%</code>'),
-    t('User accounts: <code>user/%</code>'),
-    t('Taxonomy terms: <code>taxonomy/term/%</code>'),
-    t('Comments: <code>comment/%</code>'),
-  );
-  $description_list = theme('item_list', array('items' => $items));
-
   $form['path'] = array(
     '#title' => t('Path'),
     '#field_prefix' => url('', array('absolute' => TRUE)),
     '#type' => 'textfield',
     '#default_value' => $layout->getPath(),
     '#required' => TRUE,
-    '#description' => '<p>' . t('Use the percent symbol ("%") to indicate a placeholder in the path. Some common examples include:') . '</p>' . $description_list,
     '#access' => !$layout->isDefault(),
+    '#description' => t('Use the percent symbol ("%") to indicate a placeholder in the path.'),
+  );
+
+  $description_items = array(
+    t('Content: <code>node/%</code>'),
+    t('User accounts: <code>user/%</code>'),
+    t('Taxonomy terms: <code>taxonomy/term/%</code>'),
+    t('Comments: <code>comment/%</code>'),
+  );
+  $description_list = theme('item_list', array('items' => $description_items));
+  $form['path_details'] = array(
+    '#type' => 'details',
+    '#summary' => t('Placeholder examples'),
+    '#details' => t('Some common examples of placeholders include:') . $description_list,
+    '#attributes' => array('class' => array('description')),
```

Comment:
We need to exclude this entire form element if this is a default layout:

```suggestion
    '#attributes' => array('class' => array('description')),
    '#access' => !$layout->isDefault(),
```

Otherwise you end up with this "rogue" fieldset:

![image](https://user-images.githubusercontent.com/2423362/147738340-817ceacc-21c7-4c64-a635-dcb3043bf634.png)